src/main/scala/edu/berkeley/ce/rockslicing/Delaunay.scala

```diff
@@ -23,7 +23,7 @@ object Delaunay {
     case class EdgeAnihilationSet(s: Set[IEDGE]) {
       def add(e: IEDGE): EdgeAnihilationSet = {
         if( s.contains(e) ) {
-          printf(s"FOUND ${e} NOT REVERSED *********************\n")
+//          printf(s"FOUND ${e} NOT REVERSED *********************\n")
```

Comment:
We don't particularly need this one - it just provides information on what's happening during the triangulation process. It's something we can turn back on later if we need to during debugging, but there haven't been any issues related to this thus far.